code/modules/antagonists/cult/runes.dm

```diff
@@ -222,38 +224,37 @@ structure_check() searches for nearby cultist structures required for the invoca
 /obj/effect/rune/convert/invoke(list/invokers)
 	if(rune_in_use)
 		return
+
 	var/list/myriad_targets = list()
-	var/turf/T = get_turf(src)
-	for(var/mob/living/M in T)
-		if(!IS_CULTIST(M))
-			myriad_targets |= M
+	var/turf/our_turf = get_turf(src)
+
+	for(var/mob/living/victim in our_turf)
+		if(!IS_CULTIST_OR_CULTIST_MOB(victim))
+			myriad_targets += victim
+
 	if(!length(myriad_targets))
 		fail_invoke()
 		log_game("Offer rune failed - no eligible targets.")
 		return
+
 	rune_in_use = TRUE
 	visible_message(span_warning("[src] pulses blood red!"))
-	var/oldcolor = color
 	color = RUNE_COLOR_DARKRED
-	var/mob/living/L = pick(myriad_targets)
-
-	var/mob/living/F = invokers[1]
-	var/datum/antagonist/cult/C = F.mind.has_antag_datum(/datum/antagonist/cult,TRUE)
-	var/datum/team/cult/Cult_team = C.cult_team
-	var/is_convertable = is_convertable_to_cult(L, C.cult_team)
-	if(L.stat != DEAD && is_convertable)
-		invocation = "Mah'weyh pleggh at e'ntrath!"
-		..()
-		if(is_convertable)
-			do_convert(L, invokers, Cult_team)
+	var/mob/living/selected_victim = pick(myriad_targets)
+	var/datum/team/cult/cult_team =  locate(/datum/team/cult) in GLOB.antagonist_teams
```

Comment:
Locating in global teams is weird, the old code was fine (https://github.com/tgstation/tgstation/pull/80583/files#diff-03b369e5e6c74160bc7eb992d2b7fc282e6f07d67d6669ef6d5e7f25458dcc9f)
```dm
var/mob/living/new_convertee = pick(myriad_targets)
var/mob/living/first_invoker = invokers[1]
var/datum/antagonist/cult/first_invoker_datum = first_invoker.mind.has_antag_datum(/datum/antagonist/cult)
var/datum/team/cult/cult_team = first_invoker_datum.get_team()
```

Reply:
I initially needed to do this incase my invoker didn’t have a mind